package.json

```diff
@@ -44,6 +45,7 @@
     "react-helmet": "^6.1.0",
     "react-intl": "^5.8.0",
     "react-markdown": "^4.3.1",
+    "react-moment": "^1.0.0",
```

Comment:
I think that it is not a good idea including _Moment_ if WebApp team is not already using it.

Reply:
I did rollback of  moment